kernels/etherlink/xtask/src/main.rs

```diff
@@ -0,0 +1,301 @@
+// SPDX-FileCopyrightText: 2025 Nomadic Labs <contact@nomadic-labs.com>
+//
+// SPDX-License-Identifier: MIT
+
+use anyhow::{Context, Result, bail};
+use clap::{Arg, Command};
+use std::env;
+use std::path::{Path, PathBuf};
+use xshell::{Shell, cmd};
+
+const ETHERLINK_DIR: &str = "kernels/etherlink";
+const RISCV_DIR: &str = "src/riscv";
+const SANDBOX_BIN: &str = "riscv-sandbox";
+
+const TX_COUNT: &str = "200";
+const DEFAULT_ROLLUP_ADDRESS: &str = "sr163Lv22CdE8QagCwf48PWDTquk6isQwv57";
+const DEFAULT_INBOX: &str = "src/riscv/assets/etherlink-erc20-inbox.json";
+
+#[derive(Debug, Default)]
+struct BenchConfig {
+    static_inbox: bool,
+    native: bool,
+    tracing: bool,
+    data_dir: Option<PathBuf>,
+}
+
+fn main() -> Result<()> {
+    let matches = Command::new("xtask")
+        .about("Etherlink benchmarking tool")
+        .subcommand(
+            Command::new("bench")
+                .about("Run Etherlink benchmark")
+                .arg(
+                    Arg::new("static")
+                        .short('s')
+                        .long("static")
+                        .help("Use static inbox")
+                        .action(clap::ArgAction::SetTrue),
+                )
+                .arg(
+                    Arg::new("native")
+                        .short('n')
+                        .long("native")
+                        .help("Run natively")
+                        .action(clap::ArgAction::SetTrue),
+                )
+                .arg(
+                    Arg::new("tracing")
+                        .short('t')
+                        .long("tracing")
+                        .help("Produce kernel run trace")
+                        .action(clap::ArgAction::SetTrue),
+                ),
+        )
+        .get_matches();
```

Comment:
Could you use the deriving strategy for composing the command?

See the [tutorial here](https://docs.rs/clap/latest/clap/_derive/_tutorial/index.html).